kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml

```diff
@@ -40,6 +40,12 @@
     content: "{{ core_vault_sunbird_sso_publickey }}"
   when: release_name == "learner" or release_name == "lms" or release_name == "groups" or release_name == "knowledgemw"
 
+- name: Create the token pubic key file for ML Services
```

Comment:
This is not required. Previous step does the same. Add your service in when clause.

Reply:
Sure

Reply:
ML Services will store the key file in a different way which is bit different from sunbird service.

For only ML Service Charts this will be applied

Reply:
Alritght!